src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -2670,3 +2670,30 @@ _mongoc_cse_is_enabled (mongoc_client_t *client)
 }
 
 #endif /* MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION */
+
+
+const char *
+mongoc_client_encryption_get_crypt_shared_version (
+   const mongoc_client_encryption_t *enc)
+{
+#ifdef MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION
+   return _mongoc_crypt_get_crypt_shared_version (enc->crypt);
+#else
+   BSON_UNUSED (enc);
+   return NULL;
+#endif
+}
+
+const char *
+mongoc_client_get_crypt_shared_version (const mongoc_client_t *const client)
```

Comment:
Since `mongoc_client_get_crypt_shared_version` is declared in `mongoc-client.h`, would it be preferable to define this in `mongoc-client.c` for consistency? I know it doesn't much matter once libmongoc is compiled, but I can't think of another place where we've done it this way.

Reply:
This is mostly just to colocate CSE-related code, and since this file already pulls a file-level dependency on libmongocrypt, whereas `mongoc-client.c` does not.